gui/src/connectors/redux/reducers/notifications.js

```diff
@@ -26,7 +29,19 @@ export default (state = initialState, action = {}) => {
       return Object.assign(
         {},
         state,
-        {messages: [SIGN_IN_FAIL_MSG]}
+        {errors: [{detail: SIGN_IN_FAIL_MSG}]}
+      )
+    case RECEIVE_CREATE_ERROR:
```

Comment:
How do you feel about `*_ERROR` actions clearing `*_SUCCESS` actions an vice versa? Currently if you successfully create something and then get an error it shows both the success + error message.

<img width="1043" alt="screen shot 2018-09-05 at 8 19 27 pm" src="https://user-images.githubusercontent.com/680789/45133375-ad1eb580-b149-11e8-843b-34a40d31639a.png">